src/components/color-picker/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import style from './style.styl';
+
+export default class Dropdown extends Component {
```

Comment:
this should be `ColorPicker` or `Color`, if you need to extend Dropdown you can do so here